### ISSUE TITLE
invalid copyright Greg Valure Natural Docs detected

### ISSUE THREAD

**Issue by rakeshbalusa** [nexB/scancode-toolkit#168]
OS : windows 7 - 64 bit
I've updated my scancode to the latest version available.

RPM file download URL - `http://bay.uchicago.edu/centos-vault/6.4/cr/x86_64/Packages/libreoffice-sdk-doc-4.0.4.2-9.el6.x86_64.rpm`

issue file path : `libreoffice-sdk-doc-4.0.4.2-9.el6.x86_64.rpm\usr\lib64\libreoffice\sdk\docs\cpp\ref\search\search.js`

The original text in that file is:

```
// Search script generated by doxygen
// Copyright (C) 2009 by Dimitri van Heesch.

// The code in this file is loosly based on main.js, part of Natural Docs,
// which is Copyright (C) 2003-2008 Greg Valure
// Natural Docs is licensed under the GPL.

```

`scancode output` : 

```
Path                                     What   Start Line  End Line        Info
C:/doc/temporary/delete/search.js   Copyright   4   6       Copyright (c) 2003-2008 Greg Valure Natural Docs
```

In the above output "Natural Docs" is not a part of the copyright.

**Comment by pombredanne** [comment]
@rakeshbalusa can you split this in two tickets? The copyright and the license issues are unrelated and should be each theit own bugs.

**Comment by pombredanne** [comment]
This has been fixed in #790 

The returned data are:
```
"copyrights": [
        {
          "statements": [
            "Copyright (c) 2009 by Dimitri van Heesch."
          ],
          "holders": [
            "Dimitri van Heesch."
          ],
          "authors": [],
          "start_line": 1,
          "end_line": 2
        },
        {
          "statements": [
            "Copyright (c) 2003-2008 Greg Valure"
          ],
          "holders": [
            "Greg Valure"
          ],
          "authors": [],
          "start_line": 4,
          "end_line": 6
        }
      ]
```

Thank you!
